library/cc/stream_callbacks.h

```diff
@@ -32,9 +36,21 @@ class EnvoyHttpCallbacksAdapter {
 public:
   EnvoyHttpCallbacksAdapter(ExecutorSharedPtr executor, StreamCallbacksSharedPtr callbacks);
 
+  envoy_http_callbacks as_envoy_http_callbacks();
+
 private:
+  static void* dispatch_on_headers(envoy_headers headers, bool end_stream, void* context);
```

Comment:
Maybe `c_on_headers`?

Reply:
Or maybe not - I was thinking these would be the raw function pointers to go on the struct, but that doesn't look to be the case from the implementation.

Reply:
changed these all to `c_...`, hope that's ok!

Reply:
although i noticed that maybe they're still the same in engine.h, let me go back and fix that.